applications/vanilla/models/class.categorymodel.php

```diff
@@ -1206,15 +1417,15 @@ public static function getAncestors($categoryID, $checkPermissions = true, $incl
             }
             $Max--;
 
+            if ($category['CategoryID'] == -1) {
+                break;
+            }
+
             if ($checkPermissions && !$category['PermsDiscussionsView']) {
```

Comment:
It doesn't look like `$category['PermsDiscussionsView']` is always set by the time this check is made.  Currently triggers a notice:

> Undefined index: PermsDiscussionsView